jscomp/others/js_date.ml

```diff
@@ -0,0 +1,111 @@
+type t
+
+external valueOf : t -> float = "" [@@bs.send]
+(** returns the primitive value of this date, equivalent to getTime *)
+
+external make : unit -> t = "Date" [@@bs.new]
+(** returns a date representing the current time *)
+
+external fromFloat : float -> t = "Date" [@@bs.new]
+external fromString : string -> t = "Date" [@@bs.new]
+let fromString s =
+  let date = fromString s in
```

Comment:
can we leave such error handling later, I have some ideas in #1219 
It would be nice we have `external` FFIs first, wrapper can be delayed for a while